### DISCUSSION TITLE
Event Templates

### DISCUSSION THREAD

**Discussion by reditechoz** [liedekef/events-made-easy#620] [Q&A] [answered]
Hi

I am using Default event list format to display a list of the next 4 upcoming events and it is displaying fine.

I now want to have a different template for a whole page of all upcoming events.  Where do I go to create a template?  I can see the instructions on how to refer to a template in the shortcode but can't find instructions on where to actually create the template.

Thanks

**Comment by liedekef** [comment, marked as answer]
you go in the EME submenu called "Templates".
